tests/test_crawl.py

```diff
@@ -97,8 +97,8 @@ def test_retry_dns_error(self):
 
     @defer.inlineCallbacks
     def test_start_requests_bug_before_yield(self):
+        crawler = self.runner.create_crawler(BrokenStartRequestsSpider)
         with LogCapture('scrapy', level=logging.ERROR) as l:
-            crawler = self.runner.create_crawler(BrokenStartRequestsSpider)
```

Comment:
Yep, the tests passed, probably it's better to leave it as it was in master. Thanks for the catch!